include/posix/threads.h

```diff
@@ -38,10 +38,6 @@
 
 #include <pthread.h>
 
-#ifndef PTHREAD_MUTEX_NORMAL
```

Comment:
IIRC I had put this in for a specific system that did not define it, but I cannot find it anymore.  I added the entire file because compilers were very slow to support C11 threads.  They do, meanwhile, so I am considering removing the entire file.  Would that work for FreeBSD?

Reply:
Yes it would. :)